src/apps/project-settings/ProjectSettings.css

```diff
@@ -28,6 +28,8 @@
   max-width: 65ch;
   padding: 20px 0;
   z-index: 100;
+  height: calc(100vh - 230px);
```

Comment:
I can see that breaking in various ways. (Browser differences, people having different font sizes, etc.) Is this necessary? I can help clean up the CSS later. But can't test locally right now (the staging backend doesn't have the  `projects.is_locked` column yet).